### PR TITLE
Add Card component and add them to the homepage

### PR DESCRIPTION
**Pull request by markgaze**

### CODE REVIEW

**Comment by markgaze** [comment]
#### 📝Changed routes:


| [**/**](https://somedevsdo-blog-2n9s6krrw-markgaze.vercel.app/) |
|:-:|
|<a href="https://somedevsdo-blog-2n9s6krrw-markgaze.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rmROUSKcOwRdSiJn0rRuAj_6HHoaArPKb1r6AdjQ5rJrQoL6j-7t_lihMx2lha4k0Q40R6qA5nhriIelfiduIQ.png">(view full size)</a> |







Commit 92091571efb24f9571f7fc76414d30f319b182fe (https://somedevsdo-blog-2n9s6krrw-markgaze.vercel.app).

**Comment by davidn0rman** [comment]
Like it! Assume we don't mind too much for today that we don't have the 2 then 3 columns on the homepage? Just get it in?

**Comment by markgaze** [comment]
> Like it! Assume we don't mind too much for today that we don't have the 2 then 3 columns on the homepage? Just get it in?

Yeah, I reckon so. We can make it better over time but this is a start 🙂

File: components/Card/Card.module.scss [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.link {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.card {
+  box-shadow: 0 4px 20px 0 var(--shadow);
```
Comment (davidn0rman) [inline components/Card/Card.module.scss:7]: px not rem?

File: components/Card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <div className={styles.header}>
+            <ImageWithPlaceholder
+              layout="fill"
+              objectFit="cover"
+              src={`/posts/featured/${featuredImage}`}
+            />
+          </div>
+          <div className={styles.details}>
+            <span className={styles.postDetails}>
+              <p className={styles.category}>{category}</p> -{" "}
```
Comment (davidn0rman) [inline components/Card/Card.tsx:28]:
```suggestion
              <p className={styles.category}>{category}</p>{" - "}
```
Pedantic comment, but I feel this is cleaner?

File: components/Card/Card.module.scss (hunk 1 above)
Comment (markgaze) [reply components/Card/Card.module.scss:7]: Good spot, I just took the values from Figma directly 🙂

File: components/Card/Card.tsx (hunk 2 above)
Comment (markgaze) [reply components/Card/Card.tsx:28]: I don't even remember typing this so I'm assuming Prettier did it, yours is better though so will see if it'll accept it!

File: components/Card/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          <div className={styles.header}>
+            <ImageWithPlaceholder
+              layout="fill"
+              objectFit="cover"
+              src={`/posts/featured/${featuredImage}`}
+            />
+          </div>
+          <div className={styles.details}>
+            <span className={styles.postDetails}>
+              <p className={styles.category}>{category}</p> {" - "}
```
Comment (davidn0rman) [inline components/Card/Card.tsx:28]: Hmm, guessing prettier didn't like this. Think this might come out as `[space][space][hyphen][space]`? 😢

Comment (markgaze) [reply components/Card/Card.tsx:28]: Looks like it just sacks off all the spaces anyway 🤷🏼‍♂️

**Review by davidn0rman — APPROVED** [review]

**Pull request merged**